stat/roc.go

```diff
@@ -126,3 +128,62 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 
 	return tpr, fpr, cutoffs
 }
+
+// TOC returns the Total Operating Characteristic for the classes provided
+// and the minimum and maximum bounds for the TOC.
+//
+// The input classes should be sorted according to the rank variable.
+// SortWeightedLabeled can be used to sort classes together with the rank
+// variable. The returned ntp values can be interpreted as true positives
```

Comment:
together with weights by the rank variable

Reply:
Yes. Out of practice.